sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -285,7 +329,18 @@ static void *_event_loop(void *args)
                 reply.content.value = -ENOTSUP;
                 msg_reply(&msg, &reply);
                 break;
-
+            case GNRC_IPV6_EXT_HANDLE_NEXT_HDR:
```

Comment:
This is a direct reply of a thread using plain msg passing when parsing an extension from a received IPv6 packet is finished.
The `*handle` is used as an iteration message by the thread to point to the next header to be processed, which is eventually dispatched in `gnrc_ipv6_ext_demux()`.
The IPv6 thread needs to wait forwarding the IPv6 packet further until all extensions has been processed by all interested threads.